requirements.txt

```diff
@@ -0,0 +1,4 @@
+PY34=3.4.5
+PY35=3.5.2
+PY36=3.6.0
+PY37=3.7.0
```

Comment:
maybe we use `3.7.1` which is the latest currently?

Reply:
Fixed.